echo2_config.cc

```diff
@@ -25,7 +25,7 @@ class Echo2ConfigFactory : public NamedNetworkFilterConfigFactory {
     return ProtobufTypes::MessagePtr{new Envoy::ProtobufWkt::Empty()};
   }
 
-  std::string name() override { return "echo2"; }
+  const std::string name() const override { return "echo2"; }
```

Comment:
Why is this `const std::string()`? It should either be `std::string` or `const std::string&` I think.

Reply:
I didn't want the filter to own its name string. Fine dropping const from std::string if you think that's better.

Reply:
Well, `const std::string` for a return value doesn't do anything so it should be dropped IMHO.

Reply:
Done.